docs/troubleshooting.rst

```diff
@@ -0,0 +1,23 @@
+Troubleshooting
+###############
+
+.. _troubleshooting:
+
+We are doing the best we can so you do not have to read this section.
+
+In case you do, please find the common problems you may have below.
+
+If you do not find a solution to your problem here, please
+:ref:`ask for help <communication_channels>`!
+
+
+Module object has no attribute 'register_json'
+==============================================
+
+Kinto uses PostgreSQL ``JSONBin`` feature, which allow Kinto to
```

Comment:
Kinto uses *the* JSONBin feature of PostgreSQL, which is used to store native JSON objects efficiently. Support for this feature was added in PostgreSQL 9.4.